packages/react-components/src/molecules/__tests__/EventInfo.test.tsx

```diff
@@ -29,6 +29,28 @@ it('truncates long event titles', () => {
   expect(getByRole('heading', { level: 3 }).textContent).toMatch(/…$/i);
 });
 
+it('renders event thumbnail', () => {
+  const { getByAltText } = render(
+    <EventInfo
+      {...props}
+      thumbnail={'https://placeholder/40x40'}
+      title={'blablablha'}
+    />,
+  );
+
+  expect(getByAltText(/thumbnail/i).getAttribute('src')).toEqual(
+    'https://placeholder/40x40',
+  );
+});
+
+it('renders placeholder event thumbnail', () => {
+  const { getByText } = render(
+    <EventInfo {...props} title={'blablablha'.repeat(100)} />,
```

Comment:
explicit `thumbnail={undefined}` - we don't know what the example data looks like

Reply:
```suggestion
    <EventInfo {...props} thumbnail={undefined} title={'blablablha'.repeat(100)} />,
```